apps/examples/client-example/api/openapi.json

```diff
@@ -304,6 +304,13 @@
             },
             "default": {}
           },
+          "groups": {
+            "type": "array",
```

Comment:
I strongly suggest that we limit the number of groups for an event to a single one, instead of a list in order to avoid multiple ways to achieve the same effect by different configuration criteria. In case an event needs to be enabled in multiple deployments, it needs to be put in a group that is enables in the comma-separated list in hopeit_server command line.